backend/app/models/settings_types.py

```diff
@@ -0,0 +1,13 @@
+from enum import Enum
+
+
+class AntiRecommenderType(str, Enum):
```

Comment:
Put these in separate files.

Reply:
All types have been moved to separate files.